app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name, age):
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
```

Comment:
better not to change `cls` attribute through object even if this attribute is mutable

Reply:
suggestions how  else to add elements to the dictionary on initialization?

Reply:
`        Person.people[self.name] = self`